jenkins_integration/jenkins_integration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+//For logging in terminus
+header('Content-Type: text/plain; charset=UTF-8');
+//Create curl post request to hit the Jenkins webhook
+$secrets = json_decode(file_get_contents($_SERVER['HOME'] . '/files/private/secrets.json'));
```

Comment:
There's an example of [better error checking](https://github.com/pantheon-systems/quicksilver-examples/pull/37/files#diff-f5effc2f08b43ae434669c8275d5615dR117) in #37.